build/license_check.sh

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+# Don't set -e, want to capture the output of missing files.
+
+# Print versions
+go version
+
+# Install license headers checker.
+# Switch directories to avoid changing go.mod and go.sum files (see https://stackoverflow.com/a/57313319).
+(cd / && go get -u github.com/google/addlicense)
+
```

Comment:
Do you need to switch back to the directory containing our code/configs?

Reply:
The () cause the commands to execute in a subshell, so I don't have to switch back.

Reply:
nice